interface/wx/mediactrl.h

```diff
@@ -225,6 +227,16 @@ class wxMediaEvent : public wxNotifyEvent
     wxActiveXContainer documentation.
 
 
+    @beginStyleTable
+    @style{wxMC_NO_AUTORESIZE}
+           By default, the control will automatically adjust its size to
+           exactly fit the size of a loaded video as soon as a video is loaded.
+           If this flag is given, the control will not change its size
+           automatically and must be done manually (if desired) using Layout().
```

Comment:
I think this would read a tiny bet better like this (but please feel free to ignore me if this was a conscious choice rather than just accidental omission):
```suggestion
           automatically and it must be done manually (if desired) using Layout().
```